frontend/src/features/properties/map/acquisition/AcquisitionContainer.tsx

```diff
@@ -53,7 +67,22 @@ export const AcquisitionContainer: React.FunctionComponent<IAcquisitionContainer
       header={<AcquisitionHeader acquisitionFile={acquisitionFile} />}
       footer={null}
     >
-      <></>
+      <>
+        {hasClaim(Claims.ACQUISITION_EDIT) && acquisitionFile !== undefined ? (
+          <Button variant="success" onClick={() => setIsShowingPropertySelector(true)}>
+            <MdLocationPin size={'2.5rem'} />
+            Edit properties
+          </Button>
+        ) : null}
```

Comment:
I know this is placeholder code but here we check the user has ACQUISITION_EDIT claim. However the corresponding code in Research File checks for RESEARCH_ADD - any reason for the discrepancy? I lean towards thinking that research file logic has a bug. 

Or should we check for ADD || EDIT ? That way if a user has only ADD or EDIT claims they can still see the Edit Properties button.

Reply:
Sorry, disregard my previous comment. This button only shows for UPDATE so we should check for UPDATE claim - right?

Reply:
Yes, and update = edit.